cmd/minikube/cmd/kubectl.go

```diff
@@ -84,6 +104,33 @@ var kubectlCmd = &cobra.Command{
 	},
 }
 
+func installKubectl(binary, version, path, prefix string) {
+	bindir := filepath.Join(prefix, "bin")
+	installpath := filepath.Join(bindir, binary)
+	glog.Infof("Installing %s (%s)", installpath, version)
+
+	data, err := ioutil.ReadFile(path)
+	if err != nil {
+		exit.WithError("Failed to read kubectl", err)
+	}
+	err = os.MkdirAll(bindir, 0755)
+	if err != nil {
+		exit.WithError("Failed to create directory", err)
+	}
+	err = ioutil.WriteFile(installpath, data, 0755)
+	if err != nil {
+		if os.IsPermission(err) {
+			// Permission denied (common error) is *not* a crash...
+			console.Fatal("%s: %v", "Failed to write kubectl", err)
+			os.Exit(exit.Software)
+		}
+		exit.WithError("Failed to write kubectl", err)
+	}
+}
+
 func init() {
+	kubectlCmd.Flags().BoolVar(&kubectlInstallMode, "install", false, "Install kubectl and exit")
+	kubectlCmd.Flags().StringVar(&kubectlInstallPrefix, "prefix", defaultPrefix, "Installation prefix")
```

Comment:
won't this default prefix be a problem for windows?

Reply:
Probably, but haven't used anything but MSYS myself.

Reply:
Yeah, I'm not a windows user as well. It worries me that downloading `kubeclt` currently works on all platforms, and now considering this default prefix it will break in windows. Does it make sense to only use the prefix when in non-windows systems? It basically does the previous behavior for windows, and we add an issue for someone else to implement this support.

Reply:
I can add another prefix (already have a suffix...), just not sure what it would be